src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,58 @@
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { useEffect, useState } from 'react';
+
+import { Errors } from '../types/Errors';
+import { getPeople } from '../api';
+import { Person } from '../types';
 
 export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [peopleFromServer, setPeopleFromServer] = useState<Person[]>([]);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(response => {
+        if (!response.length) {
+          setErrorMessage(Errors.noPeople);
+        } else {
+          setPeopleFromServer(response);
+        }
+      })
+      .catch(() => setErrorMessage(Errors.wentWrong))
+      .finally(() => setIsLoading(false));
+  }, []);
+
   return (
     <>
       <h1 className="title">People Page</h1>
 
       <div className="block">
         <div className="columns is-desktop is-flex-direction-row-reverse">
           <div className="column is-7-tablet is-narrow-desktop">
-            <PeopleFilters />
+            {!isLoading && <PeopleFilters />}
           </div>
 
           <div className="column">
             <div className="box table-container">
-              <Loader />
-
-              <p data-cy="peopleLoadingError">Something went wrong</p>
+              {isLoading ? (
+                <Loader />
+              ) : (
+                <PeopleTable peopleFromServer={peopleFromServer} />
+              )}
 
-              <p data-cy="noPeopleMessage">There are no people on the server</p>
+              {errorMessage === Errors.wentWrong && (
+                <p data-cy="peopleLoadingError">{Errors.wentWrong}</p>
+              )}
 
-              <p>There are no people matching the current search criteria</p>
+              {errorMessage === Errors.noPeople && (
+                <p data-cy="noPeopleMessage">{Errors.noPeople}</p>
+              )}
 
```

Comment:
The condition `errorMessage === Errors.noMatches` is used to display a message, but the `Errors.noMatches` is never set in the code. Ensure that this error condition is handled appropriately in the data fetching or filtering logic if it is intended to be used.